docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+Working with Lights
+###################
+
+:summary: This tutorial demonstrates the creation and manipulation of LightSetups in Habitat-sim.
+
+.. contents::
+    :class: m-block m-default
+
+A **LightSetup** consists of a set of LightInfo structures defining the common configuration
+of a set of point lights used to render objects in a scene. Once defined and registered,
+a LightSetup can be assigned to any subset of objects in the scene, including the scene asset itself.
+
+Each **LightInfo** structure in a LightSetup defines the `color`, `position`, and `LightPositionModel` of a single point light source.
+The LightPositionModel defines the coordinate frame of the light and can be set to any of:
+
+.. code:: python
+
+    habitat_sim.gfx.LightPositionModel.CAMERA
+    habitat_sim.gfx.LightPositionModel.GLOBAL
+    habitat_sim.gfx.LightPositionModel.OBJECT
+
+Each LightSetup is registered in the simulator via a unique key. Two default lighting setups are pre-defined:
+
+
+- **habitat_sim.gfx.DEFAULT_LIGHTING_KEY**
+
+  8 white, GLOBAL lights at scene bounding box corners.
+
+- **habitat_sim.gfx.NO_LIGHT_KEY**
+
+  0 lights (for Flat shaded assets).
+
+
+Additional custom setups can be created and registered via the **simulator.set_light_setup** function:
+
+.. code:: python
+
+    sim.set_light_setup(new_light_setup, "my_custom_lighting_key")
+
+Any existing LightSetup can be queried with **simulator.get_light_setup**:
+
+.. code:: python
+
+    custom_light_setup = sim.set_light_setup(n"my_custom_lighting_key")
+
+
+`Working with Lights Example:`_
```

Comment:
Similar comment as above -- could we state what goal is this lights example trying to accomplish?

Reply:
```suggestion
`Working with Lights Example`_
```

Typographically I don't like the colon at the end (no other heading has it), but I feel the title doesn't really work either (maybe `Basic Setup` or something like that, at least?).